CRM/CiviAwards/Service/AwardApplicationContactAccess.php

```diff
@@ -138,7 +138,7 @@ public function getReviewAccess($contactId, $applicationId, AwardReviewPanelCont
 
     foreach ($visibilitySettings as $visibilitySetting) {
       $caseStatusMatch = array_intersect($visibilitySetting['application_status'], $caseStatus) || empty($visibilitySetting['application_status']);
-      $caseTagMatch = !empty($caseTags) && (array_intersect($visibilitySetting['application_tags'], $caseTags) || empty($visibilitySetting['application_tags']));
+      $caseTagMatch = (empty($visibilitySetting['application_tags']) || (!empty($caseTags) && array_intersect($visibilitySetting['application_tags'], $caseTags)));
```

Comment:
We really don't need the first opening curly bracket and the last curly brackets `(` `)`